qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -142,22 +149,31 @@ def calibrations(self) -> Calibrations:
         return self._cals
 
     @classmethod
-    def _default_experiment_options(cls):
+    def _default_experiment_options(cls) -> Options:
         """Default values for a calibration experiment.
 
         Experiment Options:
             result_index (int): The index of the result from which to update the calibrations.
             group (str): The calibration group to which the parameter belongs. This will default
                 to the value "default".
-
         """
         options = super()._default_experiment_options()
-
-        options.result_index = -1
-        options.group = "default"
-
+        options.update_options(result_index=-1, group="default")
         return options
 
+    @classmethod
+    def _default_transpile_options(cls) -> Options:
+        """Return empty default transpile options as optimization_level is not used."""
+        return Options()
+
+    def set_transpile_options(self, **fields):
+        """Add a warning message."""
+        warnings.warn(
+            f"Transpile options are not used in {self.__class__.__name__ }. "
+            "If your experiment has overridden `_transpiled_circuits` and needs "
```

Comment:
This sounds to me not like a warning for end users. Please keep the first line and move this line to `.. note::` in the method doc.

Reply:
Done here: [88cf040](https://github.com/Qiskit/qiskit-experiments/pull/952/commits/88cf040af989c832bf8c413612ba7a552227dc00)